test/uk/gov/hmrc/tradergoodsprofilesrouter/controllers/GetRecordsControllerSpec.scala

```diff
@@ -69,6 +73,8 @@ class GetRecordsControllerSpec extends PlaySpec with MockitoSugar with GetRecord
       when(getRecordsService.fetchRecord(any, any, any)(any))
         .thenReturn(Future.successful(Right(getResponseDataWithAdviceStatus())))
 
+      when(appConfig.hawkConfig.getRecordsUrl).thenReturn("/url")
+
       val result = sut.getTGPRecord("GB123456789001", recordId)(
         FakeRequest().withHeaders(validHeaders: _*)
       )
```

Comment:
I think her it would be good verify that GetRecordsService.fetchRecord  call the appConfig.hawk.getRecordUrl function